server/repository/src/migrations/v1_01_12/location_movement_triggers.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
     sql!(
         connection,
         r#"
-        ALTER TYPE changelog_table_name ADD VALUE 'location_movement';
+        ALTER TYPE changelog_table_name ADD VALUE IF NOT EXISTS 'location_movement';
```

Comment:
just because I was running the update to get my change in, and this was troubling me.

Reply:
I downgraded my database version to get them to run 🙄